src/views/forms/TradeForm.tsx

```diff
@@ -388,7 +395,26 @@ export const TradeForm = ({
 
               {needsAdvancedOptions && <AdvancedTradeOptions />}
 
-              {alertContent && <AlertMessage type={alertType}>{alertContent}</AlertMessage>}
+              {alertContent && (
+                <AlertMessage type={alertType}>
+                  {shouldPromptUserToPlaceLimitOrder ? (
+                    <Styled.MessageWithPrompt>
+                      {alertContent}
+                      {
+                        <Styled.IconButton
+                          iconName={IconName.Arrow}
+                          shape={ButtonShape.Circle}
+                          action={ButtonAction.Navigation}
+                          size={ButtonSize.XSmall}
+                          onClick={() => setCurrentTradeType(TradeTypes.LIMIT)}
+                        ></Styled.IconButton>
+                      }
+                    </Styled.MessageWithPrompt>
+                  ) : (
+                    alertContent
+                  )}
+                </AlertMessage>
+              )}
```

Comment:
since `Styled.MessageWithPrompt` only adds gap when there're two elements, can we just do something like?
```
<Styled.Message>
{alertContent}
{shouldPromptUserToPlaceLimitOrder ? <Styled.IconButton .../> : null} // or just condition && IconButton?
</Styled.Message>
```